server/testutil.go

```diff
@@ -137,3 +140,21 @@ func MustWaitLeader(re *require.Assertions, svrs []*Server) *Server {
 	})
 	return leader
 }
+
+// CreateMokHandler creates a mock handler for test.
+func CreateMokHandler(re *require.Assertions, ip string) HandlerBuilder {
```

Comment:
I'm not sure why it was called mok before, is mock written wrong?

Reply:
done

Reply:
`server_test.go` and `func CreateMokHandler` also has some legacy `mok`

Reply:
ptal @lhy1024

Reply:
replace name in url